### PR TITLE
PostLink logic

### PR DESCRIPTION
**Pull request by arodu**
- separate get/post links classes
- abstractClass for common link login
- disable link logic
-  WIP PostLink logic

### CODE REVIEW

**Comment by steinkel** [comment]
Please add the POST delete link to the setRowActions https://github.com/CakeDC/cakephp-datatables/blob/feature/pagination-fixes/src/View/Helper/DatatableHelper.php#L285

File: src/View/Helper/DatatableHelper.php [hunk 1]
```diff
@@ -157,9 +158,23 @@ class DatatableHelper extends Helper
...
 
+
+
                 },
             });
         });
+
+        function withConfirm(element, message, confirmCondition) {
+            //alert("confirm message");
+            console.log(element);
```
Comment (steinkel) [inline src/View/Helper/DatatableHelper.php]: remove log traces

File: src/View/Helper/DatatableHelper.php [hunk 2]
```diff
@@ -157,9 +158,23 @@ class DatatableHelper extends Helper
...
+            console.log(message);
+            console.log(confirmCondition);
+        }
+
+        function withPostLink(button) {
+            alert("post link");
```
Comment (steinkel) [inline src/View/Helper/DatatableHelper.php]: remove

**Review by steinkel — CHANGES_REQUESTED** [review]

**Comment by arodu** [comment]
> Please add the POST delete link to the setRowActions https://github.com/CakeDC/cakephp-datatables/blob/feature/pagination-fixes/src/View/Helper/DatatableHelper.php#L285

it's here
https://github.com/CakeDC/cakephp-datatables/blob/feature/post-delete-links/src/View/Helper/DatatableHelper.php#L294-L295

**Pull request merged**